src/app/agentic/page.tsx

```diff
@@ -2,13 +2,13 @@
 import { useEffect, useRef, useState, type FormEvent } from "react";
 import { useRouter } from "next/navigation";
 import Link from "next/link";
-import {
-  CROSSMINT_BASE_URL,
-  CROSSMINT_CLIENT_API_KEY,
-  JWT_SUBJECT,
-  BASIS_THEORY_URL,
-  BASIS_THEORY_ENVIRONMENT,
-} from "@/app/consts";
+const CROSSMINT_BASE_URL = process.env.NEXT_PUBLIC_CROSSMINT_BASE_URL!;
```

Comment:
u should never need a base url if u have the api key

if u cal `validateAPIKey` from sdk it tells u what env the api key is for, and then use `environmentToCrossmintBaseURL`